public/canvas/index.js

```diff
@@ -0,0 +1,33 @@
+(function(window, $) {
+  'use strict';
+
+  var loadCanvasCustomization = function() {
+    window.CALCENTRAL = 'https://junction.berkeley.edu';
+
+    // Ensure the bCourses development and test servers are pointing to the correct
+    // CalCentral instance when a copy of production is made
+    if (window.location.origin === 'https://ucberkeley.beta.instructure.com') {
+      window.CALCENTRAL = 'https://junction-dev.berkeley.edu';
+    } else if (window.location.origin === 'https://ucberkeley.test.instructure.com') {
+      window.CALCENTRAL = 'https://junction-qa.berkeley.edu';
```

Comment:
In the URLs above, replace `junction` with `ripley`. Secondly, replace `berkeley.edu` with `ets.berkeley.edu`.